File: 2st/Riyeon/콜라츠 추측.py
def solution(num):
    answer = 0
    for i in range(1, 501):
        if num == 1:
            break
        else:
            answer += 1
            
        if num%2 == 0:
            num/=2
        else:
            num = num*3+1
    if num != 1:
        answer = -1
    return answer

File: 4st/Eunseong/콜라츠 추측.java
class Solution {
    public int solution(long num) {
        int i = 0;
        int answer = 0;
        if (i == 1) return 0;
        while(num != 1) {
            if (num % 2 == 0) {
                num /= 2;
            } else {
                num *= 3;
                num++;
            }
            i++;
        }
        if (i > 500) {
            answer = -1; 
        } else {
            answer = i;
        }
            
        return answer;
    }
}